server/package.json

```diff
@@ -11,6 +12,7 @@
   "description": "",
   "dependencies": {
     "bcrypt": "^5.0.1",
+    "body-parser": "^1.19.0",
```

Comment:
See this comment: https://github.com/shiv-chan/senpai-kohai/pull/2/files#r744211693